src/Data/Time/PreciseDuration/Format.purs

```diff
@@ -0,0 +1,81 @@
+module Data.Time.PreciseDuration.Format where
+
+import Prelude
+
+import Data.Array as Array
+import Data.Decimal (Decimal, modulo)
+import Data.Maybe (Maybe(..))
+import Data.Monoid (guard)
+import Data.Time.PreciseDuration (PreciseDuration)
+import Data.Time.PreciseDuration as PD
+
+type Format =
+  { intermediateZeroes :: Boolean
+  }
```

Comment:
This is so we could potentially have things like `leadingZeroes` and `trailingZeroes` in the future